utils.ts

```diff
@@ -67,6 +70,22 @@ export function sourceMapToLineHits(hitTracks: Map<string, SourceTrack>) {
   return sourceToLineMapping;
 }
 
+/**
```

Comment:
So because I've been struggling with how to name the argument context, I instead documented the exports in hopes to better explain it.